app/build.gradle

```diff
@@ -52,25 +54,25 @@ task printVersionName {
 
 //def LOCAL_IP = getLocalIPv4()
 def LOCAL_IP = "10.0.2.2"
-def URL_PROXY_BASE_LOCAL = "http://"+LOCAL_IP+":14000"
+def URL_PROXY_BASE_LOCAL = "http://" + LOCAL_IP + ":14000"
```

Comment:
I got an automatic formatter/lint running on all changed files when committing. will turn this off in the future.

Reply:
sure, i'll do that. will run the default(most used) formatting for the entire project

Reply:
Just to be clear I meant in a separate PR :)

It would be great if you could do that right away, then we'll revert the formatting changes from this PR (I can do that). This would make it much easier for me to understand the changes.